client/src/lib/solana/transaction.ts

```diff
@@ -38,4 +38,20 @@ export class SolidTransaction {
     const recordKey = await getKeyFromAuthority(authority);
     return SolidTransaction.getSolid(connection, recordKey);
   }
+
+  static async deactivateSolid(
+    connection: Connection,
+    payer: Account,
+    recordKey: PublicKey
```

Comment:
You can add another account param here for the authority

Reply:
Done